ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/ui/main/panel/template/ServerErrorPageTemplate.kt

```diff
@@ -1,29 +1,17 @@
 package org.jetbrains.research.tasktracker.ui.main.panel.template
 
-import org.jetbrains.research.tasktracker.config.MainTaskTrackerConfig
+import org.jetbrains.research.tasktracker.TaskTrackerPlugin
 
-// TODO: add a config with userMessage instead of DEFAULT_TEXT
-class ServerErrorPageTemplate(private val userMessage: String = defaultText) : HtmlBaseTemplate() {
-    override val content: String
-        get() = buildPageTemplate()
+class ServerErrorPageTemplate(private val userMessage: String = defaultText) : HtmlBaseFileTemplate() {
```

Comment:
I'd suggest to add a base class for an error page and make a hierarchy for them since we already have a default error page, the server error page